examples/templates/docker-image-builds/main.tf

```diff
@@ -84,7 +46,7 @@ variable "docker_image" {
 }
 
 resource "docker_volume" "home_volume" {
-  name = "coder-${data.coder_workspace.me.owner}-${lower(data.coder_workspace.me.name)}-root"
+  name = "coder-${data.coder_workspace.me.owner_id}-${lower(data.coder_workspace.me.id)}-root"
```

Comment:
This change might trigger https://github.com/coder/coder/issues/3386 for certain users who like to keep up-to-date with our templates. I think this is a change we really want, but I'd like for some warning/notifying facility to be in place before we do.

At minimum, we should list this as a breaking change in the release notes and warn of the behavior (update template + update workspace = volume loss).

Reply:
I'll add it to the release notes on the patch.

We advertise that users shouldn't stay up-to-date with our templates, so I'm not very concerned about updating.

Reply:
> We advertise that users shouldn't stay up-to-date with our templates, so I'm not very concerned about updating.

Hmm, I don't see this mentioned in `examples` or e.g. the `docker` template?

Anyway, my main motivation behind being extra careful about #3386 & co. is that I don't want our current users to lose confidence in Coder because they lost their volume (even if it's due to their own configuration/changes).

Reply:
Fair enough. I can also exclude this if you feel it's sketchy. It's not needed for the improvement.

Reply:
I'm going to exclude it for right now. It's not worth the possible tumult for users.

Reply:
> Fair enough. I can also exclude this if you feel it's sketchy. It's not needed for the improvement.

If that's ok, we could at least defer it until we make the change to all templates? Since I don't know if/when we'll have a solution for #3386, I won't ask us to defer until then. But doing it all at once minimizes the spread, at least.

Reply:
Yup, removed!